### PR TITLE
DM-34095: plot displacement and velocity from accelerometer data

### PR DESCRIPTION
**Pull request by pkubanek**
- Moved VMS files into VMS directory
- Split ChartWidgets
- CacheWidget
- trapz velocity integral
- CacheTimeWidget
- added DisplacementWidget

### CODE REVIEW

File: bin/VMSGUI [hunk 1]
```diff
@@ -47,8 +51,24 @@ class EUI(QMainWindow):
         i = 0
         for s in self.SYSTEMS:
             m = menuBar.addMenu(s)
+            m.addAction(
+                "New &raw graph",
```
Comment (teweitsai) [inline bin/VMSGUI:55]: "New & raw graph"

File: bin/VMSGUI [hunk 2]
```diff
@@ -47,8 +51,24 @@ class EUI(QMainWindow):
...
+                partial(
+                    self._addCacheWidget, i, "Raw acceleration", RawAccelerationWidget
+                ),
+            )
             m.addAction("New &box graph", partial(self.addBox, i))
-            m.addAction("New &PSD graph", partial(self.addPSD, i))
+            m.addAction(
+                "New &PSD graph", partial(self._addCacheWidget, i, "PSD", PSDWidget)
```
Comment (teweitsai) [inline bin/VMSGUI:62]: "New PSD graph"

File: bin/VMSGUI [hunk 3]
```diff
@@ -47,8 +51,24 @@ class EUI(QMainWindow):
...
+            )
+            m.addAction(
+                "New &Velocity graph",
```
Comment (teweitsai) [inline bin/VMSGUI:65]: "New Velocity graph"

File: bin/VMSGUI [hunk 4]
```diff
@@ -47,8 +51,24 @@ class EUI(QMainWindow):
...
+                partial(self._addCacheWidget, i, "Velocity", VelocityWidget),
+            )
+            m.addAction(
+                "New &Displacement graph",
```
Comment (teweitsai) [inline bin/VMSGUI:69]: "New Displacement graph"

File: python/lsst/ts/cRIOpy/VMS/CacheWidget.py [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+    @Slot(int, int, float, float)
+    def cacheUpdated(self, index, length, startTime, endTime):
+        """Process and plot data.
+
+        Parameters
+        ----------
+        index : `int`
+        length : `int`
+        startTime : `float`
+        endTime : `float`
```
Comment (teweitsai) [inline python/lsst/ts/cRIOpy/VMS/CacheWidget.py:114]: doc string?

**Review by teweitsai — APPROVED** [review]
Looks good. Please help to improve the documents. Thanks!

File: bin/VMSGUI (hunk 1 above)
Comment (pkubanek) [reply bin/VMSGUI:55]: No. & is used to mark accelerator key - R in this case.

File: bin/VMSGUI (hunk 2 above)
Comment (pkubanek) [reply bin/VMSGUI:62]: & marks accelerator key.
image

**Pull request merged**